WBooks/Books_Table/BookTableViewCell.swift

```diff
@@ -32,7 +32,7 @@ class BookTableViewCell: UITableViewCell, NibLoadable {
         authorLabel.text = book.author
         logoImageView.image = Constants.defaultImage
         
-        if let imageURL = URL(string: book.imageUrl) {
+        if let urlString = book.imageUrl, let imageURL = URL(string: urlString) {
```

Comment:
You can do `if let imageURL = book.imageUrl.flatMap({URL(string: $0)}) { ... }` if you don't need the urlString variable.
You can search what `map` and `flatMap` mean to optionals